.buildkite/ftr_security_stateful_configs.yml

```diff
@@ -120,7 +120,6 @@ enabled:
   - x-pack/platform/test/spaces_api_integration/deployment_agnostic/security_and_spaces/stateful.config_trial.ts
   - x-pack/platform/test/spaces_api_integration/deployment_agnostic/security_and_spaces/stateful.copy_to_space.config_trial.ts
   - x-pack/platform/test/spaces_api_integration/deployment_agnostic/security_and_spaces/stateful.copy_to_space.config_basic.ts
-  - x-pack/solutions/security/test/alerting_api_integration/security_and_spaces/group1/config.ts
```

Comment:
there were two tests in this group and both belong with the rest of the alerting tests, not security tests